docs/samples/tensorflow/README.md

```diff
@@ -22,7 +22,7 @@ $ kfservice.serving.kubeflow.org/flowers-sample configured
 MODEL_NAME=flowers-sample
 INPUT_PATH=@./input.json
 CLUSTER_IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-SERVICE_HOSTNAME=$(kubectl get kfservice ${MODEL_NAME} -o jsonpath='{.status.url}')
+SERVICE_HOSTNAME=$(kubectl get kfservice ${MODEL_NAME} -o jsonpath='{.status.url}' |sed -e 's/^http:\/\///g' -e 's/^https:\/\///g')
```

Comment:
I wonder if we could use xip.io to simplify this somehow?

Reply:
For this, we have to set custom domain in knative. right?

Reply:
What's going on here? Substituting http for https? I guess this is the result of going from domain -> url

Reply:
If we set our ingress to automatically upgrade http requests, this could be a viable option to just have http in all of the curl examples.